index.html

```diff
@@ -38,19 +46,235 @@ <h1 class="boldtext">
             <a href="#"><i class="fa-brands fa-twitter fa-xl iconspace"></i></a>
           </li>
           <li>
-            <a href="#"><i class="fa-brands fa-linkedin-in fa-xl iconspace"></i></a>
+            <a href="#"
+              ><i class="fa-brands fa-linkedin-in fa-xl iconspace"></i
+            ></a>
           </li>
           <li>
-            <a href="#"><i class="fab fa-medium fa-xl iconspace iconspace"></i></a>
+            <a href="#"
+              ><i class="fab fa-medium fa-xl iconspace iconspace"></i
+            ></a>
           </li>
           <li>
-            <a href="#"><i class="fa-brands fa-github fa-xl iconspace "></i></a>
+            <a href="#"><i class="fa-brands fa-github fa-xl iconspace"></i></a>
           </li>
           <li>
             <a href="#"><i class="fab fa-angellist fa-xl iconspace"></i></a>
           </li>
         </ul>
       </div>
     </section>
+
+    <!-- works section starts here -->
+    <section class="works-section-container">
+      <div class="card-container">
+        <div class="img-container">
+          <img  class="card-image" src="images/works_section_images/img1.png" alt="">
+        </div>
+        <div class="content-container">
+          <h2 class="project-title">Tonic</h2>
+          <div class="frame2">
```

Comment:
- [ ] [OPTIONAL] You can use `article` tag instead for more semantic HTML, kindly try changing it